tasks/cron.yml

```diff
@@ -47,3 +47,34 @@
   with_items:
     - { type: "core", day: "2,4,6" }
     - { type: "kerneloops", day: "1,3,5" }
+
+- name: cron - faf find-crashfn core
+  cron:
+    name: "cron for faf find-crashfn for core"
+    user: faf
+    job: "faf find-crashfn -p core"
+    special_time: daily
+    state: present
+
+- name: cron - faf find-crashfn kerneloops
+  cron:
+    name: "cron for faf find-crashfn for kerneloops"
+    user: faf
+    job: "faf find-crashfn -p kerneloops"
+    minute: 15
+    hour: "*/3"
+    state: present
+
+- name: retrace symbols
```

Comment:
Wasn't this already merged in https://github.com/abrt/ansible-role-faf/pull/11 ?